developer-docs/server-installation/current_release_tags_n_jenkins_jobs.md

```diff
@@ -17,25 +17,25 @@ This page provides information on the release tags available for Sunbird install
 | sunbird-auth              | release-1.15.0 | Build/Core/Keycloak |
-| SunbirdEd-portal          | release-2.6.0 | Build/Core/Player |
-| sunbird-content-plugins   | release-2.6.0 | Build/Plugins/ContentPlugins |
-| sunbird-content-player   | release-2.6.0 | Build/Plugins/ContentPlayer |
-| sunbird-content-editor   | release-2.6.0 | Build/Plugins/ContentEditor |
-| sunbird-collection-editor   | release-2.6.0 | Build/Plugins/CollectionEditor |
-| sunbird-generic-editor  | release-2.6.0 | Build/Plugins/GenericEditor |
+| SunbirdEd-portal          | release-2.8.0 | Build/Core/Player |
+| sunbird-content-plugins   | release-2.8.0 | Build/Plugins/ContentPlugins |
+| sunbird-content-player   | release-2.8.0 | Build/Plugins/ContentPlayer |
+| sunbird-content-editor   | release-2.8.0 | Build/Plugins/ContentEditor |
+| sunbird-collection-editor   | release-2.8.0 | Build/Plugins/CollectionEditor |
+| sunbird-generic-editor  | release-2.8.0 | Build/Plugins/GenericEditor |
 | sunbird-telemetry-service | release-2.1.0 | Build/Core/Telemetry |
-| sunbird-course-service    | release-2.6.0 | Build/Core/Lms |
+| sunbird-course-service    | release-2.8.0 | Build/Core/Lms |
 | enc-service | release-1.1.1 | Build/Core/EncService |
 | cert-service | release-1.1.1 | Build/Core/Cert |
-| sunbird-data-pipeline     | release-2.6.0| Build/DataPipeline/DataPipeline, All jobs under ArtifactUpload/{{env}}/DataPipeline, All jobs under Deploy/{{env}}/DataPipeline, All jobs under OpsAdministration/{{env}}/DataPipeline, All jobs under Provision/{{env}}/DataPipeline|
+| sunbird-data-pipeline     | release-2.8.0| Build/DataPipeline/DataPipeline, All jobs under ArtifactUpload/{{env}}/DataPipeline, All jobs under Deploy/{{env}}/DataPipeline, All jobs under OpsAdministration/{{env}}/DataPipeline, All jobs under Provision/{{env}}/DataPipeline|
 | secor                     | secor-0.25 | Build/DataPipeline/Secor  |
-| sunbird-analytics         | release-2.6.0 | Build/DataPipeline/Analytics|
-| sunbird-learning-platform | release-2.6.0 | All jobs under Build/KnowledgePlatform, All jobs under ArtifactUpload/{{env}}/KnowledgePlatform, All jobs under Deploy/{{env}}/KnowledgePlatform, All jobs under OpsAdministration/{{env}}/KnowledgePlatform, All jobs under Provision/{{env}}/KnowledgePlatform |
-| shared-lib   | release-2.6.0-shared-lib | This will be set in Manage Jenkins -> Global Pipeline Libraries -> Default Version |
+| sunbird-analytics         | release-2.8.0 | Build/DataPipeline/Analytics|
+| sunbird-learning-platform | release-2.8.1 | All jobs under Build/KnowledgePlatform, All jobs under ArtifactUpload/{{env}}/KnowledgePlatform, All jobs under Deploy/{{env}}/KnowledgePlatform, All jobs under OpsAdministration/{{env}}/KnowledgePlatform, All jobs under Provision/{{env}}/KnowledgePlatform |
+| shared-lib   | release-2.7.0-shared-lib | This will be set in Manage Jenkins -> Global Pipeline Libraries -> Default Version |
```

Comment:
rest all of the release numbers is 2.8.0

Reply:
That is fine. we have to give that 2.7.0